### ISSUE TITLE
Attempt unsigned downloads if AWS credentials not found

### ISSUE THREAD

**Issue by micahjsmith** [HDI-Project/piex#6]
* D3M Pipeline Explorer version: 0.1.1
* Python version: 3.6.8
* Operating System: Linux eb804d3cf037 4.9.125-linuxkit #1 SMP Fri Sep 7 08:20:28 UTC 2018 x86_64 GNU/Linux

### Description

piex fails to download publicly-available pipelines if the user has not populated an AWS credentials files. This should succeed without any configuration.

### What I Did

I try to run the following in an environment without AWS credentials

```
import piex.explorer
ex = piex.explorer.S3PipelineExplorer('ml-pipelines-2018')
ex.get_pipelines()
```

But I get the following credential-related error.
```
Traceback (most recent call last):       
  File "./main.py", line 238, in <module>                                                                                        
    main()                      
  File "./main.py", line 226, in main                                                                                            
    make_table_1()                           
  File "./main.py", line 117, in make_table_1                                                                                    
    df = load_pipelines_df(ex=ex)
  File "/usr/local/lib/python3.6/site-packages/funcy/calc.py", line 44, in wrapper                                               
    value = memory[key] = func(*args, **kwargs)
  File "./main.py", line 40, in load_pipelines_df                                                                                
    df = ex.get_pipelines() 
  File "/usr/local/lib/python3.6/site-packages/piex/explorer.py", line 280, in get_pipelines                                     
    df = self._load_table('pipelines')
  File "/usr/local/lib/python3.6/site-packages/piex/explorer.py", line 252, in _load_table
    df = self._get_table(table_name)                                                                                             
  File "/usr/local/lib/python3.6/site-packages/piex/explorer.py", line 234, in _get_table
    obj = s3.get_object(Bucket=self.bucket, Key=key)
  File "/usr/local/lib/python3.6/site-packages/botocore/client.py", line 320, in _api_call                                       
    return self._make_api_call(operation_name, kwargs)
  File "/usr/local/lib/python3.6/site-packages/botocore/client.py", line 611, in _make_api_call                                  
    operation_model, request_dict)
  File "/usr/local/lib/python3.6/site-packages/botocore/endpoint.py", line 102, in make_request                                  
    return self._send_request(request_dict, operation_model)
  File "/usr/local/lib/python3.6/site-packages/botocore/endpoint.py", line 132, in _send_request                                 
    request = self.create_request(request_dict, operation_model)
  File "/usr/local/lib/python3.6/site-packages/botocore/endpoint.py", line 116, in create_request                                
    operation_name=operation_model.name)
  File "/usr/local/lib/python3.6/site-packages/botocore/hooks.py", line 356, in emit                                             
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/botocore/hooks.py", line 228, in emit                                             
    return self._emit(event_name, kwargs)
  File "/usr/local/lib/python3.6/site-packages/botocore/hooks.py", line 211, in _emit                                            
    response = handler(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/botocore/signers.py", line 90, in handler                                         
    return self.sign(operation_name, request)
  File "/usr/local/lib/python3.6/site-packages/botocore/signers.py", line 157, in sign                                           
    auth.add_auth(request)
  File "/usr/local/lib/python3.6/site-packages/botocore/auth.py", line 425, in add_auth                                          
    super(S3SigV4Auth, self).add_auth(request)
  File "/usr/local/lib/python3.6/site-packages/botocore/auth.py", line 357, in add_auth                                          
    raise NoCredentialsError
botocore.exceptions.NoCredentialsError: Unable to locate credentials                                                             
make: *** [run] Error 1
```

**Comment by micahjsmith** [comment]
if no credentials, can use

``` python
import boto3
import botocore
import botocore.config
s3 = boto3.client('s3', config=botocore.config.Config(signature_version=botocore.UNSIGNED))
```

**Comment by csala** [comment]
I think an even better option would be to just remove boto3 altogether and just use plain HTTP:

```
url = 'https://{}.s3.amazonaws.com/{}'.format(bucket, key)
response = urllib.request.urlopen(url)
bytes_io = io.BytesIO(response.read())
```

**Comment by micahjsmith** [comment]
Totally up to you. Perhaps there would be a situation you envision in which people would want to use piex on pipelines that are not publicly available? if not, then boto probably not necessary. Otherwise, its capabilities could be helpful.

**Comment by csala** [comment]
Fixed in v0.2.0
